src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,19 @@
-import React from 'react';
 import './MoviesList.scss';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[]
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <div className="movies">
+    <ul>
+      {movies.map(movie => (
+        <li>
+          <MovieCard movie={movie} key={movie.imdbId} />
+        </li>
+      ))}
+    </ul>
```

Comment:
Need to fix it a little.You added 'ul.' Maybe correct for semantics, but the grid stopped working and all cards are placed in a column. 
![image](https://user-images.githubusercontent.com/105741133/188852664-de6fc856-bf56-4749-9419-229c74f0d551.png)